### PR TITLE
Fix #599: Move communities to separate page

### LINKED ISSUE #599: Move communities to separate page.
Move list of communities to separate page /kugs. Link it from main page.
Search all links together

### PR DESCRIPTION
**Pull request by IRus**
Make sure that you are making pull request against [`main`](https://github.com/KotlinBy/awesome-kotlin/tree/main) branch. Pull requests against `readme` branch will not be accepted, because all manual changes to this branch will be overridden by changes from `main` branch. Consult [contributing.md](https://github.com/KotlinBy/awesome-kotlin/blob/main/.github/contributing.md) for details.

Checklist: 

- [x] Is this pull request against the branch `main`?

### CODE REVIEW

File: app-frontend/app/src/components/bar/Navigation.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  }),
+  bar: style({
+    backgroundColor: "rgba(242, 242, 242, 0.7)",
+    $nest: {
+      "& > *": {
+        boxSizing: "border-box"
+      }
+    }
+  })
+};
```
Comment (madhead) [inline app-frontend/app/src/components/bar/Navigation.tsx:34]: Why did the styles go to the `.tsx` file, and not `.less`?

File: app-frontend/app/src/links.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import {Category} from "./model";
+
+const data: Category[] = require("../links.json");
```
Comment (madhead) [inline app-frontend/app/src/links.ts:3]: Could the type from `model.tsx` be reused here? (`type Links = Category[]`).

File: app-frontend/webpack.config.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ module.exports = function (options = {}) {
 
   const APP_DIR = path.resolve(__dirname, "app");
 
-  console.log(`
+  console.error(`
```
Comment (madhead) [inline app-frontend/webpack.config.js:14]: Why `error`?

**Review by madhead — APPROVED** [review]

File: app-frontend/app/src/components/bar/Navigation.tsx (hunk 1 above)
Comment (IRus) [reply app-frontend/app/src/components/bar/Navigation.tsx:34]: Some styles initially was done in less, later project switched to typestyle. It was easier to maintain only required classes, and reference them from components. Also there are was idea to do easy programmatically-defined themes (light, dark)

File: app-frontend/webpack.config.js (hunk 3 above)
Comment (IRus) [reply app-frontend/webpack.config.js:14]: Cause `log` breaks default flow of https://github.com/webpack/analyse
It's easy work around, to not manually change stats.json

File: app-frontend/app/src/links.ts (hunk 2 above)
Comment (IRus) [reply app-frontend/app/src/links.ts:3]: Yes

**Pull request merged**